packages/ui/sd.config.ts

```diff
@@ -32,6 +32,13 @@ StyleDictionary.extend({
       files: [
         {
           destination: 'dist/theme.css',
+          // This filter function will remove any design tokens that are marked with a deprecatedStateToken flag
+          filter: (token) => {
```

Comment:
I am bit curious what would happen if we do not filter out the old tokens with `_`. Theoretically, the generated variable names should stay the same and should be fine?

Reply:
If we don't filter them out style dicitionary will create two entries with the same variable, it doesn't actually break anything to have them both exist because they both have the same value, but this keeps the generated css a little cleaner.